lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -256,16 +260,21 @@ export class RefreshTokenClient extends BaseClient {
                 request.correlationId
             )(refreshTokenRequest);
         } catch (e) {
-            if (
-                e instanceof InteractionRequiredAuthError &&
-                e.subError === InteractionRequiredAuthErrorCodes.badToken
-            ) {
-                // Remove bad refresh token from cache
-                this.logger.verbose(
-                    "acquireTokenWithRefreshToken: bad refresh token, removing from cache"
+            if (e instanceof InteractionRequiredAuthError) {
+                this.performanceClient?.addFields(
```

Comment:
If we log this right before each request we can do it in one place and have the data for all requests